governance/communications/call_to_action.md

```diff
@@ -0,0 +1,14 @@
+# Call to Action template
```

Comment:
Generally i think we use `-` delimeters instead of `_`

Reply:
Done